site/content/docs/user/quick-start.md

```diff
@@ -17,6 +17,8 @@ This guide covers getting started with the `kind` command.
 You can either install kind with `GO111MODULE="on" go get sigs.k8s.io/kind@v0.7.0` or clone this repo 
 and run `make build` from the repository.
 
+You also need to make sure that [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) is installed.
```

Comment:
tiny bit: s/need to/should/

KIND does not require kubectl to function technically, but you will not be able to use the demo commands or follow most of the guide without it.

Reply:
added.